### ISSUE TITLE
Add support for controlling overall audio

### ISSUE THREAD

**Issue by banool** [kyleneideck/BackgroundMusic#611]
I would love it if BackgroundMusic supported changing the overall volume. I have a device that doesn't allow you to do that, and BackgroundMusic helps a bit, but I have to go and change each app manually. I'd love to have a "master" slider that applies at the end to modify every app's volume.

**Comment by elarochejoubert** [comment]
I would love that too !!

**Comment by Kickflipkid687** [comment]
Does this application let you quickly pause/play sources as well?  I'm trying to find a good way to do that yet...

If i had 1+ youtube tabs/sources open and am listening to iTunes/Music lets say.  If i want to quickly pause 1 and play another.  

Ideally a popup, under the mouse, where if I hit a hotkey, a popup is there, and I can quickly play/pause.  Basically like the Play Icon in the top menu of MacOS Monterey, but under your mouse...  or more quickly accessible..

**Comment by ndroftheline** [comment]
> Does this application let you quickly pause/play sources as well? I'm trying to find a good way to do that yet...
> 
> If i had 1+ youtube tabs/sources open and am listening to iTunes/Music lets say. If i want to quickly pause 1 and play another.
> 
> Ideally a popup, under the mouse, where if I hit a hotkey, a popup is there, and I can quickly play/pause. Basically like the Play Icon in the top menu of MacOS Monterey, but under your mouse... or more quickly accessible..

i don't think this comment applies to this query, you can add a separate feature request issue about this.

i didn't realise the lack of an overall/master volume level was working as intended. presumably this is not an easy thing to fix. would it make sense to start a bounty for this feature somewhere? or is this some fundamental limitation? thanks

**Comment by denalena** [comment]
I too would love this feature! +1
